packages/amplify-nodejs-function-template-provider/resources/lambda/crud/package.json.ejs

```diff
@@ -11,6 +11,6 @@
   },
   "devDependencies": {
     "@types/aws-lambda": "^8.10.92",
-    "aws-sdk": "^2.1113.0"
+    "aws-sdk": "^2.1230.0"
```

Comment:
I think that this is no-op because the carat symbol would already resolve ^2.1113.0 to the same version as ^2.1230.0

Reply:
While true in a semantic versioning sense, in practice it's not necessarily true if dependencies are locked. We should standardize to a single version so that only a single version of the `aws-sdk` is needed.